src/Orchard/ContentManagement/Utilities/LazyField.cs

```diff
@@ -11,17 +11,23 @@ public class LazyField<T> {
             set { SetValue(value); }
         }
 
-        public void Loader(Func<T, T> loader) {
+        public void Loader(Func<T> loader) {
             _loader = loader;
         }
 
+        [Obsolete]
+        public void Loader(Func<T, T> loader)
+        {

```

Comment:
Oh god... when I back home I will fix formatting.
BTW. I think Orchard should use StyleCop it is extremely usefull.